tests/unit/ml_handlers/test_statsforecast.py

```diff
@@ -198,3 +231,8 @@ def test_auto_model_selection(self, mock_handler):
         # check against ground truth
         mindsdb_result = result_df.iloc[:, -1]
         assert np.allclose(mindsdb_result, package_predictions)
+
+        describe_result = self.run_sql("describe proj.modelx")
+        assert len(describe_result["accuracies"][0]) > 1
```

Comment:
Same as in the comment above, would be nice to have a short explanation of the expected data structure.